### PR TITLE
chore(litmus-portal): Updating the loginpage and homepage

### PR DESCRIPTION
**Pull request by amityt**
Signed-off-by: amityt <amitkumar.das@mayadata.io>

**What this PR does / why we need it**: Updated the login page with form component and added the recent changes in the Homepage

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Checklist**

* [ ] Does this PR have a corresponding GitHub issue?
* [ ] Have you included relevant README for the chaoslib/experiment with details?
* [ ] Have you added debug messages where necessary? 
* [ ] Have you added task comments where necessary? 
* [ ] Have you tested the changes for possible failure conditions?
* [ ] Have you provided the positive & negative test logs for the litmusbook execution?
* [ ] Does the litmusbook ensure idempotency of cluster state?, i.e., is cluster restored to original state?
* [ ] Have you used non-shell/command modules for Kubernetes tasks?
* [ ] Have you (jinja) templatized custom scripts that is run by the litmusbook, if any? 
* [ ] Have you (jinja) templatized Kubernetes deployment manifests used by the litmusbook, if any?
* [ ] Have you reused/created util functions instead of repeating tasks in the litmusbook?
* [ ] Do the artifacts follow the appropriate directory structure? 
* [ ] Have you isolated storage (eg: OpenEBS) specific implementations, checks? 
* [ ] Have you isolated platform (eg: baremetal kubeadm/openshift/aws/gcloud) specific implementations, checks?  
* [ ] Are the ansible facts well defined? Is the scope explicitly set for playbook & included utils?
* [ ] Have you ensured minimum/careful usage of shell utilities (awk, grep, sed, cut, xargs etc.,)?
* [ ] Can the limtusbook be executed both from within & outside a container (configurable paths, no hardcode)?
* [ ] Can you suggest the minimal resource requirements for the litmusbook execution?
* [ ] Does the litmusbook job artifact carry comments/default options/range for the ENV tunables?
* [ ] Has the litmusbooks been linted? 

**Special notes for your reviewer**:

### CODE REVIEW

File: litmus-portal/frontend/src/pages/LoginPage/index.tsx [hunk 1]
```diff
@@ -1,62 +1,122 @@
...
+              id="login-form"
+              className={classes.root}
+              autoComplete="on"
+              onSubmit={(event) => {
+                event.preventDefault();
+                handleForm();
+              }}
+            >
+              <div className={classes.inputDiv}>
+                <Paper className={classes.inputArea}>
```
Comment (gdsoumya) [inline litmus-portal/frontend/src/pages/LoginPage/index.tsx:63]: Is the Paper component necessary? Try [this](https://material-ui.com/components/text-fields/)

File: litmus-portal/frontend/src/pages/LoginPage/index.tsx [hunk 2]
```diff
@@ -1,62 +1,122 @@
...
+                    required
+                    onChange={(e) =>
+                      setAuthData({
+                        username: e.target.value,
+                        password: authData.password,
+                      })
+                    }
+                  />
+                </Paper>
+                <Paper className={classes.inputArea}>
```
Comment (gdsoumya) [inline litmus-portal/frontend/src/pages/LoginPage/index.tsx:80]: Same as above

File: litmus-portal/frontend/src/pages/LoginPage/index.tsx (hunk 1 above)
Comment (amityt) [reply litmus-portal/frontend/src/pages/LoginPage/index.tsx]: [x] Updated

Comment (amityt) [reply litmus-portal/frontend/src/pages/LoginPage/index.tsx]: * [x] Updated

File: litmus-portal/frontend/src/pages/LoginPage/index.tsx (hunk 2 above)
Comment (amityt) [reply litmus-portal/frontend/src/pages/LoginPage/index.tsx]: * [x] Updated

File: litmus-portal/frontend/src/pages/LoginPage/index.tsx [hunk 3]
```diff
@@ -1,62 +1,113 @@
...
+  password: string;
+}
+
 const LoginPage = () => {
   const classes = useStyles();
+  const [authData, setAuthData] = useState<authData>({
+    username: '',
+    password: '',
+  });
+  const handleForm = () => {
```
Comment (rajdas98) [inline litmus-portal/frontend/src/pages/LoginPage/index.tsx:17]: Add one line before the function

File: litmus-portal/frontend/src/pages/LoginPage/index.tsx [hunk 4]
```diff
@@ -1,62 +1,113 @@
...
+    data.append('username', username);
+    data.append('password', password);
+    fetch('/login', {
+      method: 'POST',
+      body: data,
+    }).then((response) => {
+      response.text();
+    });
+  };
   return (
```
Comment (rajdas98) [inline litmus-portal/frontend/src/pages/LoginPage/index.tsx:33]: Add one line before the return statement

Comment (amityt) [reply litmus-portal/frontend/src/pages/LoginPage/index.tsx:35]: Added the changes @rajdas98 PTAL

**Pull request merged**